scripts/zones/Toraimarai_Canal/npcs/_4pc.lua

```diff
@@ -2,16 +2,14 @@
 -- Area: Toraimarai Canal
 -- NPC:  Marble Door
 -- Involved In Windurst Mission 7-1
--- @pos 132 12 -19 169 169
+-- @pos 132 12 -19 169
 -----------------------------------
 package.loaded["scripts/zones/Toraimarai_Canal/TextIDs"] = nil;
-require("scripts/zones/Toraimarai_Canal/TextIDs");
```

Comment:
Removed the require but still nil-ing it out. If no textIDs are used, remove'em both. If any are used, we need both.